src/data/partners.yaml

```diff
@@ -0,0 +1,86 @@
+# CBET Business Partners
+- medwrench:
```

Comment:
this is how we will add new partners from now on and add the image in the `/src/data/logos` directory.